modules/osd-monitoring-default-monitoring-components.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * monitoring/osd-understanding-the-monitoring-stack.adoc
+
+[id="default-monitoring-components_{context}"]
+= Default monitoring components
+
+By default, the {product-title} monitoring stack includes these components:
+
+.Default monitoring stack components
+[options="header"]
+|===
+
+|Component|Description
+
+|Cluster Monitoring Operator
+|The Cluster Monitoring Operator (CMO) is a central component of the monitoring stack. It deploys and manages Prometheus instances, the Thanos Querier, the Telemeter Client, and metrics targets and ensures that they are up to date. The CMO is deployed by the Cluster Version Operator (CVO).
```

Comment:
from the monitoring architecture, CMO manages Prometheus Operator instance,  Prometheus Opeartor instance manages Prometheus and Alertmanager instances.
we could change
` It deploys and manages Prometheus instances`
to 
`It deploys and manages Prometheus Operator instances`
or
`It deploys and manages Prometheus and  Alertmanager instances`

Reply:
Done.